### DISCUSSION TITLE
Request to fix issue with name of one currency

### DISCUSSION THREAD

**Discussion by xrhodium** [bisq-network/bisq#5455] [General]
Hello,
there is a one request which is waiting for a long time...

https://github.com/bisq-network/bisq/pull/5416

There is only a name change. => String text change only.

Can you review it and merge it please?

Thank you.

**Comment by ghost** [comment]
Hello,
is there somebody who can do review and merge of this pull request 

https://github.com/bisq-network/bisq/pull/5416

It is a trivial change. :) 

Maybe @ripcurlx please?
